findimports.py

```diff
@@ -1017,8 +1033,12 @@ def main(argv=None):
                         ignore_stdlib_modules=args.ignore_stdlib)
     if args.write_cache:
         g.writeCache(args.write_cache)
+
     if args.condense_to_packages:
-        g = g.packageGraph(args.packagelevel)
+        g = g.packageGraph(args.packagelevel, False)
+    elif args.condense_to_packages_externals:
+        g = g.packageGraph(args.packagelevel, True)
```

Comment:
Personal style rule: never pass a `True`/`False`  using positional argument syntax; always use keyword argument syntax to make it clear what options this thing enables or disables.